wodles/aws/tests/test_tools.py

```diff
@@ -161,6 +162,22 @@ def test_arg_valid_iam_role_duration_raises_exception_when_invalid_duration_prov
         aws_tools.arg_valid_iam_role_duration(arg_string)
 
 
+def test_arg_validate_security_lake_auth_params():
```

Comment:
Only the first `if` clause is being tested:
https://github.com/wazuh/wazuh/blob/e64f2eff1fa35b0901b18a1079671a46deb088cb/wodles/aws/aws_tools.py#L275-L283

The [parametrize](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html) decorator could be used to test all possible options, take a look at this example:
https://github.com/wazuh/wazuh/blob/e64f2eff1fa35b0901b18a1079671a46deb088cb/wodles/aws/tests/test_aws_s3.py#L68-L85